e2-studio-star-rx72n-firmware/src/rx_stack_monitor.c

```
@@ -0,0 +1,302 @@
+ * @since Version 1.0.0
+ *
+ * @par NASA Power of 10 Compliance:
+ * - Rule 5: ✓ 2 preconditions, 2 postconditions
+ * - Rule 7: ✓ tx_thread_stack_error_notify() return value checked and mapped
+ */
+rx_err_t rx_stack_monitor_init(void)
+{
+    /* Precondition: must be called before threads start running */
+    /* (Enforced by contract: caller is tx_application_define()) */
+
+    const UINT tx_status = tx_thread_stack_error_notify(internal_stack_overflow_handler);
```

Clang (14.0.6)

[warning] variable 'tx_status' is not initialized

(cppcoreguidelines-init-variables)